astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
The problem is that with this function, say, `NUMPY_LT_1_9` will be `True` is numpy is not present.  It wouldn't seem this function's business to decide what should happen on an import error.

I guess I'm not sure, though, why you do the try/except at all. Is that necessary to resolve a problem? I ask mostly because I cannot see how whatever is resolved could have worked before, where numpy had to be imported just as much. (I do see why you need the string name, as this avoids importing numpy before the test is actually needed, but when the test has to be done, I don't see why you would hide the possible absence of numpy, surely something else will break too).